discovery/syncer.go

```diff
@@ -1378,6 +1356,11 @@ func (g *GossipSyncer) ApplyGossipFilter(filter *lnwire.GossipTimestampRange) er
 		defer returnSema()
 
 		for _, msg := range newUpdatestoSend {
+			if err := g.maybeAddMsgDelay(); err != nil {
```

Comment:
Looking at the diff again, I think ti'll just move this call into `sendToPeerSync`.